back/spec/services/sanitization_service_spec.rb

```diff
@@ -62,6 +62,15 @@
     end
 
     it "allows links to pass through when link feature is enabled" do
+      input = <<~HTML
+        <a href="https://www.google.com" target="_blank" rel="noreferrer noopener">Link</a>
+      HTML
+      features = [:link]
+      expect(service.sanitize(input, features)).to eq input
+    end
+
+    #[TODO]
```

Comment:
If you want to add a test for desired behaviour (which I suspect is the case here?), you could adapt the test to check the behaviour you would like and set `pending: true`. The test would fail but will not cause CI to fail. This the mechanism that rspec provides for these situations.

Reply:
Thanks for the tip, fixed !